salaTest/testisovist.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2020 Petros Koutsolampros
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "catch.hpp"
+#include "salalib/mgraph.h"
+
+TEST_CASE("Simple Isovist") {
+
+    const float EPSILON = 0.001;
+
+    // simple plan for isovist. dot as the origin
+    //  _ _ _ _
+    // |     . |
+    // |    _ _|
+    // |   |
+    // |_ _|
+
+    std::vector<Line> planLines = {
+        Line(Point2f(1, 1), Point2f(1, 3)), //
+        Line(Point2f(1, 3), Point2f(3, 3)), //
+        Line(Point2f(3, 3), Point2f(3, 2)), //
+        Line(Point2f(3, 2), Point2f(2, 2)), //
+        Line(Point2f(2, 2), Point2f(2, 1)), //
+        Line(Point2f(2, 1), Point2f(1, 1))  //
```

Comment:
Could probably lose the empty comments at the end of the line?

Reply:
Oh, these are for lining up the Line() calls with the auto-formatter